backend/.env

```
@@ -1 +1,13 @@
 
+# Supabase Configuration
+SUPABASE_URL=https://ejtrcbdoatgxsvmsqnzi.supabase.co
+SUPABASE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImVqdHJjYmRvYXRneHN2bXNxbnppIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTgyODQzODMsImV4cCI6MjA3Mzg2MDM4M30.fhdy-22__1JjM0i23IG4hKBaJXJp17eieJhBW3ZHe4M
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SUPABASE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)